services/datamanager/builtin/file_deletion.go

```diff
@@ -34,11 +34,14 @@ func shouldDeleteBasedOnDiskUsage(ctx context.Context, captureDirPath string, lo
 		return false, nil
 	}
 	if usedSpace < fsThresholdToTriggerDeletion {
-		logger.Warnf("disk not full enough, exiting used space: %f", usedSpace)
 		return false, nil
 	}
 	// Walk the dir to get capture stats
-	return exceedsDeletionThreshold(ctx, captureDirPath, float64(usage.Size()), logger)
+	shouldDelete, err := exceedsDeletionThreshold(ctx, captureDirPath, float64(usage.Size()), logger)
+	if err != nil && !shouldDelete {
+		logger.Warn("Disk nearing capacity but capture dir is not large enough, file deletion will not run!")
```

Comment:
Useful logging add! Similar to other suggestion for user readability of what we're measuring, perhaps `logger.Warn("Disk nearing capacity but data capture directory is below %f of that size, file deletion will not run", captureDirToFSUsageRatio)`

Reply:
done